eoscostarica/guias.eoscostarica.io
# Feat(frontend): Mob and Pair Programming 191

### GH Issue

Resolve #191 

### Steps to test
1.  Go to Metodos de Desarrollo de Software
2.  Check content is right.

#### CheckList
- [ ] Add Mob Programming
- [ ] Add Pair Programming
